Source/Forms/Gen4/Wild4.cpp

```diff
@@ -465,6 +466,16 @@ void Wild4::searcherEncounterIndexChanged(int index)
             break;
         }
 
+        if ((currentProfile->getVersion() & Game::HGSS) != Game::None && index > 2)
```

Comment:
Change index to using the encounter type we get above. Will be easy to read when mark what encounters we care about instead of wondering what does "index > 2" mean

Reply:
done